server/socketHandlers/getPlayers.js

```diff
@@ -0,0 +1,22 @@
+import { formatOutput } from '../formatters/socketHandlerFormatter';
+
+const EVENT_TYPES = {
+  ON: 'getPlayers',
+  EMIT: 'resultGetPlayers'
+}
+
+const getPlayers = (socket, socketNameSpace) => {
+  socket.on(EVENT_TYPES.ON, (roomId) => {
+    socketNameSpace.adapter.clients([roomId], (err, players) => {
+      if(err){
+        socket.emit(EVENT_TYPES.EMIT, formatOutput({error: err}));
+        return;
+      }
+      socket.emit(EVENT_TYPES.EMIT, formatOutput({data: players}));
+    });
+  });
+};
+
+
+
+export { getPlayers };
```

Comment:
In order to activate this function, please add this into `server/socketHandlers/index.js`.

```js
...
import { disconnect } from './disconnect';
import { getPlayers } from './getPlayers';  // this is what you created in this PR.

const setSocketEventHandler = (socket, socketNameSpace) => {

  const handlers = [
    ...
    disconnect,
    getPlayers  //  this is what you created in this PR.
  ];

  handlers.forEach(handler => {
    handler(socket, socketNameSpace);
  });
};
```